larq/quantized_variable.py

```diff
@@ -360,4 +364,7 @@ def _as_graph_element(self):
 tf.register_tensor_conversion_function(
     QuantizedVariable, QuantizedVariable._dense_var_to_tensor
 )
-ops.register_dense_tensor_like_type(QuantizedVariable)
+try:
+    ops.register_dense_tensor_like_type(QuantizedVariable)
```

Comment:
So this is for `tensorflow<=2.2.0` and no longer needed because of the new `core.Tensor` baseclass in `>=2.3.0rc0`?

Reply:
Correct